src/main/java/com/xpandit/plugins/xrayjenkins/task/XrayImportBuilder.java

```diff
@@ -534,6 +526,10 @@ private UploadResult uploadResults(FilePath workspace,
 			if(BuilderUtils.isEnvVariableUndefined(this.testExecKey)
 					&& StringUtils.isNotBlank(sameTestExecutionKey)
 					&& "true".equals(importToSameExecution)){
+				if(targetEndpoint.getName().contains(MULTIPART)){
```

Comment:
This condition it's a bit weird to understand, can you refactor this to a method called: isMultipartEndpoint() or something like that?